actions/class.Runner.php

```diff
@@ -128,11 +130,16 @@ protected function getServiceContext()
     {
         if (!$this->serviceContext) {
             $testExecution = $this->getSessionId();
+            /** @var DeliveryExecution $execution */
             $execution = $this->getServiceLocator()->get(DeliveryExecutionService::SERVICE_ID)->getDeliveryExecution($testExecution);
+            $userIdentifier = common_session_SessionManager::getSession()->getUser()->getIdentifier();
```

Comment:
We can use SessionService here:
`$userIdentifier = $this->getServiceLocator()->get(SessionService::SERVICE_ID)->getCurrentUser()->getIdentifier();`